static/app/views/settings/organizationAuthTokens/authTokenRow.tsx

```diff
@@ -107,19 +107,19 @@ export function OrganizationAuthTokensAuthTokenRow({
             title={t(
               'You must be an organization owner, manager or admin to revoke a token.'
             )}
-            disabled={canRevoke}
+            disabled={!!revokeToken}
           >
             <Confirm
-              disabled={!canRevoke || isRevoking}
-              onConfirm={() => revokeToken(token)}
+              disabled={!revokeToken || isRevoking}
+              onConfirm={revokeToken ? () => revokeToken(token) : undefined}
               message={t(
                 'Are you sure you want to revoke this token? The token will not be usable anymore, and this cannot be undone.'
               )}
             >
               <Button
                 size="sm"
-                onClick={() => revokeToken(token)}
-                disabled={isRevoking || !canRevoke}
+                disabled={isRevoking || !revokeToken}
+                data-test-id={`revoke-token-${token.id}`}
```

Comment:
instead of a `data-test-id` is there a `aria-label` we can use to target this element?